src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
```

Comment:
When wearing my client developer-API user hat, the RESTful endpoint documentation is the first thing I look for. That tells me what the true interface of the API with fewer leaky abstractions.